pushmanager/templates/pushes.html

```diff
@@ -9,6 +9,16 @@
 <ul id="action-buttons">
 	<li><button id="expand-pushes">Expand All</button></li>
 	<li><button id="collapse-pushes">Collapse All</button></li>
+		&mdash;
+	<li>
+		Show states:
```

Comment:
This looks like it's a duplicate of the lines below. Bad merge resolution, maybe?